hca/define_api.py

```diff
@@ -2,36 +2,133 @@
 
 import json, sys, os
 from io import open
+import pprint
 import requests
 from .parser import get_parser
 
 
 class API:
     """Class for interacting with the REST Api."""
 
-    def __init__(self, api_url, user):
+    def __init__(self, user, password, test=False):
```

Comment:
Yeah that makes much more sense. Wasn't sure how we were doing authentication at the beginning - I think that user, pw combo was from back then. Not sure why it got re-added.